ui/package.json

```diff
@@ -50,6 +50,7 @@
     "ttag-cli": "^1.7.1",
     "url-loader": "1.0.1",
     "webpack": "4.18.0",
+    "webpack-bundle-analyzer": "^3.3.2",
```

Comment:
The build got stuck. Im thinking if it might be caused by this dependency? 
```
 [exec]     [11] ./src/style/fonts/roboto-font/Roboto-Regular.ttf 82 bytes {0} [built]
     [exec]     [12] ./src/style/fonts/roboto-font/Roboto-Medium.ttf 82 bytes {0} [built]
     [exec]     [13] ./src/style/fonts/roboto-font/Roboto-Bold.ttf 82 bytes {0} [built]
     [exec]     [14] ./src/images/404/bg-01.jpg 82 bytes {0} [built]
     [exec]     [15] ./src/images/404/atronaut.png 82 bytes {0} [built]
     [exec]     [16] ./src/images/404/falling-01.png 82 bytes {0} [built]
     [exec]     [17] ./src/images/404/falling-02.png 82 bytes {0} [built]
     [exec]     [18] ./src/images/404/tube.png 82 bytes {0} [built]
     [exec]         + 8 hidden modules
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
The build has been terminated
````

Maybe we should remove it and see if it works? I think the dependency tries to open a web browser to show the result. I think that we can reproduce this locally by runing ´mvn install´in the root folder.